NEWS.md

```diff
@@ -1,5 +1,9 @@
 # simstudy (development version)
 
+## Minor fixes
+
+* Fixed bug in `trtAssign`
```

Comment:
Following semver.org this would be a so called 'patch' release bumping the version to 0.5.1  e.g. with `usethis::use_version("patch")`